multinet/auth/util.py

```diff
@@ -1,11 +1,22 @@
 """Utility functions for auth."""
 
 import functools
-from typing import Any, Optional, Callable
-
-from multinet.errors import Unauthorized
+import jwt
+import re
+import calendar
+from jwt.exceptions import InvalidSignatureError, ExpiredSignatureError
```

Comment:
The reason I did this is because everywhere else in the code base, we tend to do package imports (`import ...`) before more specific imports (`from ... import ...`)

Reply:
Ok. We should possibly have a more official guidance for this but no big deal right now.